SiriIntents/ContactResolver/ContactResolver.m

```diff
@@ -0,0 +1,176 @@
+// 
+// Copyright 2022 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#import "ContactResolver.h"
+@import Intents;
+#import "MXKAccountManager.h"
+
+@implementation ContactResolver
+
+- (void)resolveContacts:(nullable NSArray<INPerson *> *)contacts
+         withCompletion:(void (^)(NSArray<INPersonResolutionResult *> * _Nonnull))completion {
```

Comment:
Our objc style is to have `{` on the next line.
```suggestion
         withCompletion:(void (^)(NSArray<INPersonResolutionResult *> * _Nonnull))completion
{
```

Reply:
Thanks for pointing that out! Nowadays, I do not spend time thinking about code formatting, since the projects that I am working on usually have automatic code formatting set up. Maybe that's something that could be incorporated into this repository, even though the Objective-C code will likely be discarded once Element-X sees mass adoption.